src/sign-verify/clu_verify.c

```diff
@@ -346,7 +346,7 @@ int wolfCLU_verify_signature_rsa(byte* sig, char* out, int sigSz, char* keyPath,
             ret = BAD_FUNC_ARG;
         }
         else {
-            XFWRITE(outBuf, 1, outBufSz, s);
+            XFWRITE(outBuf, 1, XSTRLEN((char*)outBuf), s);
```

Comment:
We can not guarantee that the input/output will be a null termenated string for XSTRLEN to get the length on. In this case 'ret' returned from a successful wc_RsaSSL_Verify, should be the size of the data minus the padding.

Reply:
Switched to using `ret`. Thanks for the tip!